api/v1alpha3/awscluster_types.go

```diff
@@ -104,6 +104,11 @@ type Bastion struct {
 	// Cluster API Provider AWS will use t3.micro for all regions except us-east-1, where t2.micro
 	// will be the default.
 	InstanceType string `json:"instanceType,omitempty"`
+
+	// InstanceImage will use a specified AMI to boot the bastion. If not specified,
+	// the image will default to one picked out in public space.
+	// +optional
+	InstanceImage string `json:"instanceImage,omitempty"`
```

Comment:
```suggestion
	AMI string `json:"ami,omitempty"`
```

or should we actually have the same interface as awsmachine_types.go, i.e.

```suggestion
	AMI AWSResourceReference `json:"ami,omitempty"`
```

WDYT?

Reply:
+1 to using AMI here for naming rather than InstanceImage

@randomvariable Unless the intent is to honor all the fields of AWSResourceReference, I don't think we should add additional uses of it.

Reply:
Fair.